zephyr/Kconfig

```diff
@@ -37,4 +37,13 @@ config DMA_DOMAIN
 	  and dma_single_chan in Zephyr once it becomes more
 	  stable.
 
+config ZEPHYR_DP_SCHEDULER
+	bool "use Zephyr thread based DP scheduler"
+	default y if ACE
+	default n
+	depends on IPC_MAJOR_4
```

Comment:
You can also "depends on ZEPHYR_SOF_MODULE" to hide it from builds where Zephyr is not used.